app.rambox.ramboxce.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: app.rambox.ramboxce
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/linux-gnu/GL
+    version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-download:create
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+command: ramboxce
+tags: [opensource]
+separate-locales: false
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: ramboxce
+    buildsystem: simple
+    build-commands:
+      - install -d /app/share/ramboxce
+      - install -Dm755 ramboxce.sh /app/bin/ramboxce
+      - install -Dm644 app.rambox.ramboxce.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 app.rambox.ramboxce.desktop -t /app/share/applications
+      - ar x ramboxce.deb
+      - tar xf data.tar.xz
+      - find usr/share/icons -depth -name '*rambox.png*' -execdir bash -c 'for f; do mv -i "$f" "${f//rambox.png/app.rambox.ramboxce.png}"; done' bash {} +
```

Comment:
Instead, at toplevel:
```
rename-icon: rambox.png
copy-icon: true
```

Reply:
it's actually `rename-icon: rambox`

Definitely better.

Reply:
Turns out `copy-icon` may not be needed, since the tray icon is derived from `resources/app.asar` rather than the icon theme.